responders/Gmail/Gmail_UnblockSender.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "Gmail_UnblockSender",
+    "version": "1.0",
+    "author": "David Strassegger, OSCD",
```

Comment:
Would you mind extending it to the 'oscd.community' or '@oscd_initiative'? 
It would be more clear and help to engage more people to the next sprints (:

Reply:
Yeah no problem. I changed it to `@oscd_initiative`